### PR TITLE
client tool to get sign from compliance check service

### PR DESCRIPTION
**Pull request by ToWorld**
Add a client tool to get sign from compliance check service

### CODE REVIEW

File: pb/xcheck.proto [hunk 1]
```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package pb;
+
+import "google/api/annotations.proto";
+import "xchain.proto";
+
+service Xcheck {
+    rpc ComplianceCheck(TxStatus) returns (ComplianceCheckResponse);
```
Comment (yucaowang) [inline pb/xcheck.proto]: this is a client rpc proto, it's better keep the proto in cmd/cli.
pb folder is more like server-side proto.

File: cmd/cli/compliance_check.go [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+	if err != nil {
+		return errors.New("Fail to open serialized transaction data file")
+	}
+	tx := &pb.Transaction{}
+	err = proto.Unmarshal(data, tx)
+	if err != nil {
+		return errors.New("Fail to Unmarshal proto")
+	}
+
+	tx.Txid, err = txhash.MakeTransactionID(tx)
```
Comment (yucaowang) [inline cmd/cli/compliance_check.go]: Why here need a txid? at this time there is no signatures in txdata, so using MakeTransactionID cannot get the final txid.

File: pb/xcheck.proto [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+}
+
+message ComplianceCheckResponse {
+    Header header = 1;
+	SignatureInfo signature = 2;
```
Comment (yucaowang) [inline pb/xcheck.proto]: please format proto file using same indent characters.

Comment (ToWorld) [reply pb/xcheck.proto]: Ok, I fix it.

File: cmd/cli/compliance_check.go (hunk 2 above)
Comment (ToWorld) [reply cmd/cli/compliance_check.go]: The client should pass TxStatus to the server.

File: pb/xcheck.proto (hunk 1 above)
Comment (ToWorld) [reply pb/xcheck.proto]: Yes, your advice is better and i will fix it.

File: cmd/cli/compliance_check.go [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+	}
+	// XcheckClient
+	reply, err2 := c.XcheckClient().ComplianceCheck(ctx, txStatus)
+	if err2 != nil {
+		fmt.Println("check here new XcheckClient error", err2)
+		return err2
+	}
+	if reply.Header.Error != pb.XChainErrorEnum_SUCCESS {
+		return fmt.Errorf("Failed to post tx:%s, logid:%s", reply.Header.Error.String(), reply.Header.Logid)
+	}
```
Comment (MichelleZ) [inline cmd/cli/compliance_check.go]: Post tx ? Fix it？

**Pull request merged**